test_ai.py

```
@@ -0,0 +1,26 @@
+
+from backend.ai.agent.core import get_agent
+
+async def main():
+    agent = get_agent()
+    try:
+        # Override the agent client to ensure it's loaded correctly
+        res = await agent.process("مين حاجز النهاردة", tenant_id=1, user_id=1)
+        with open("test_ai_output.json", "w", encoding="utf-8") as f:
+            json.dump(res, f, ensure_ascii=False, indent=2)
+        print("Success! Wrote to test_ai_output.json")
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)